.gitignore

```diff
@@ -26,3 +26,6 @@ target
 
 # packaged app
 build/
+
+# files generated by execution
+settings.json
```

Comment:
- I don't have this file generate in the root of the project but in `~/.local/...`
- Now we use a settings with `.yaml`

Reply:
The file should be a yaml indeed.

The program tries to read/write the settings in the preference path (we get it with a sdl2 function). I've assumed that if the writing fail in this place, the program could at least write the settings in the current dir.

I'm not sure anymore if that's worth it to put it in the current dir when the settings folder for our application failed.